### DISCUSSION TITLE
How can I convert pdf into MusicXML?

### DISCUSSION THREAD

**Discussion by smadarsz** [w3c-cg/musicxml#472] [Q&A]
I have a score in pdf format and would like to convert it automatically to MusicXML.
Is there an API that supports this action?

**Comment by adrianholovaty** [comment]
Hi @smadarsz — PDF scanning isn't a part of the MusicXML specification, but third-party software exists for this function. It's called "optical music recognition."

For an open source solution, try [Audiveris](https://github.com/Audiveris/audiveris). I've also helped develop [a commercial version at Soundslice](https://www.soundslice.com/blog/226/pdf-and-photo-scanning-beta/) that uses artificial intelligence for high accuracy.

**Reply by smadarsz** [reply]
@adrianholovaty Thank you! Will explore these.

**Reply by kcpr** [reply]
Thank You so much for asking and for the answer. [Audiveris](https://github.com/Audiveris/audiveris) worked in my case I think, even though, looking at the interface, it seems like a bit older software. Available through Flatpak though.
Anyway I could export the score and open it in MuseScore and that's what I needed.
Bless!
